.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.enabled": true
+}
```

Comment:
Delete this file and its containing folder from your branch.

Reply:
Also add `.vscode/` to the .gitignore file.